cmd/new_test.go

```diff
@@ -35,6 +36,50 @@ func newMockTime() time.Time {
 	return time.Date(2021, 5, 22, 13, 30, 10, 5, time.UTC)
 }
 
+func TestEnvOnNew(t *testing.T) {
+	cfg := newTestConfig()
+	cfg.Post = []core.PostProcessConfig{
+		{Key: "author", Value: "{{.Env.AUTHOR}}"},
+	}
+	then.WithTempDirConfig(t, cfg)
+	reader, writer := then.WithReadWritePipe(t)
+	os.Setenv("ENVPREFIX_AUTHOR", "test author")
```

Comment:
```suggestion
	t.Setenv("ENVPREFIX_AUTHOR", "test author")
```
Be sure to use t.Setenv for test scoped env vars.

Reply:
Nice, TIL